### DISCUSSION TITLE
Macro support for release title -> *arr

### DISCUSSION THREAD

**Discussion by adapowers** [autobrr/autobrr#1828] [Ideas]
UNIT3D (and especially its popular community upload script) uses AKA for alternate titles, e.g. `Dan Da Dan AKA Dandandan.S01E08`. This causes Sonarr to reject the release as an unknown series.
As powerful macro support is already built for the Action section (stunning work), it seems like a way to go about fixing this would be to enable modification of the release name fed to Sonarr for purposes of matching. From what I can tell, it would be pretty trivial to write a macro that ditches the AKA.

**Comment by StormPooper** [comment]
This would also be useful for release groups that use numbering that does not match Sonarr, to be strip out the incorrect numbers or increment them to be correct. One additional wrinkle to consider would be the filenames within the download for downloads that aren't a single file - I think it would need to be able to match any file that matches the regex and support multiple and partial matches to allow remaining all matches, otherwise Sonarr will match when grabbed but fail when importing. Some may also need multiple macros as the folder naming may be completely different to the files within them.
